eos_data_distribution/avahi_monitor.py

```diff
@@ -33,6 +33,8 @@
 from eos_data_distribution.MDNS import ServiceDiscovery
 
 
+MAX_PEERS = 5
```

Comment:
How was this number chosen? That should be added as a (docstring) comment by it.